.gitignore

```diff
@@ -1,17 +1,22 @@
+# Configuration
+/app/config/parameters.yml
+
+# Libraries
+/bower_components
+/vendor
```

Comment:
the `/<folder>/*` format has better support in command line tools (like `ack` and `ag` for example)